src/cmds/qmgr.c

```diff
@@ -2865,7 +2870,10 @@ execute(int aopt, int oper, int type, char *names, struct attropl *attribs)
 						}
 					}
 				} else {
-					perr = pbs_manager(sp->s_connect, oper, type, pname->obj_name, attribs, NULL);
+					if ((strlen(pname->obj_name) ==0) && type ==MGR_OBJ_SCHED && oper != MGR_CMD_DELETE) {
+						perr = pbs_manager(sp->s_connect, oper, type, PBS_DFLT_SCHED_NAME, attribs, NULL);
```

Comment:
These changes are to avoid failures in PTL test case. The change will be removed once we fix in PTL tests/famework.  The entire change will be reverted. 

  
- For qmgr sched list, name is optional.  Earlier there was no concept of sched name. qmgr sched would list was returning default scheduler. Some of the PTL test cases are calling  qmgr sched list and expecting to return default scheduler. 
  Above changes would help to avoid PTL failures.